### ISSUE TITLE
NTSC output as black and white PAL-60

### ISSUE THREAD

**Issue by ghost** [b-rad-NDi/Ubuntu-media-tree-kernel-builder#48]
I purchased the Hauppauge USB-Live2 and was able to capture pure PAL and SECAM videos on linux with it without any problems. However, I have a few NTSC tapes that my VCR output as PAL-60 but the dongle doesn't seem to be able to handle them. If I switch to NTSC, I only get black and white videos. Is there any way for the capture card to handle PAL-60 signal ?

**Comment by b-rad-NDi** [comment]
I'll take a look and see if that is a supported standard in the decoder driver.

**Comment by b-rad-NDi** [comment]
PAL-60 is in the decoder driver. How are you attempting to switch the decoder to PAL-60?

Inspect current standard:
```v4l2-ctl -d /dev/video0 --get-standard```
Look for PAL-60:
```v4l2-ctl -d /dev/video0 --list-standards```
Set to PAL-60:
```v4l2-ctl -d /dev/video --set-standard 13```
Note: your video node might be different from video0 and your PAL-60 index might not be 13.

**Comment by mrpewpew** [comment]
Yes, I already tried to set it to PAL-60 using v4l2-ctl but I didn't get any signal recorded. Only setup that kind of works is NTSC but it's in black and white.

**Comment by b-rad-NDi** [comment]
I guess you found a bug in the driver. I unfortunately don't have any way to test PAL-60. You should send a mail to linux-media@vger.kernel.org describing the issue and maybe someone will have the time and means to look into it.

**Comment by mrpewpew** [comment]
Oh it would be fantastic if you could test it out, even if it's in August. In the meantime, I'll send the email. Thanks again

**Comment by mrpewpew** [comment]
Hi ! Did you manage to test it on the rpi ?

**Comment by b-rad-NDi** [comment]
Hey @mrpewpew been super busy, but I'll try and get to this next week.
